apps/web/src/pages/TermsAndConditions/TermsAndConditions.tsx

```diff
@@ -0,0 +1,856 @@
+            </TableOfContents.Section>
+            <TableOfContents.Section id={sections.accessibilityCommitment.id}>
+              <TableOfContents.Heading
+                size="h3"
+                data-h2-margin="base(x3, 0, x1, 0)"
+              >
+                {sections.accessibilityCommitment.title}
+              </TableOfContents.Heading>
+              <p data-h2-margin="base(x1 0)">
+                {intl.formatMessage(
+                  {
+                    id: "4jMHwL",
+                    defaultMessage:
+                      "The Government of Canada is committed to achieving a high standard of accessibility as defined in the <accessibilityLink>Standard on Web Accessibility</accessibilityLink> and the <optimizeLink>Standard on Optimizing Websites and Applications for Mobile Devices</optimizeLink>. In the event of difficulty using our web pages, applications or device-based mobile applications, contact us for assistance or to obtain alternative formats such as regular print, Braille or another appropriate format.",
+                    description:
+                      "Paragraph describing accessibility commitment section",
+                  },
+                  {
+                    accessibilityLink,
+                    optimizeLink,
+                  },
+                )}
+              </p>
+            </TableOfContents.Section>
+            <TableOfContents.Section id={sections.socialMedia.id}>
+              <TableOfContents.Heading
+                size="h3"
+                data-h2-margin="base(x3, 0, x1, 0)"
+              >
+                {sections.socialMedia.title}
+              </TableOfContents.Heading>
+              <p data-h2-margin="base(x1 0)">
+                {intl.formatMessage({
+                  id: "nxAw/X",
+                  defaultMessage:
+                    "This notice has been written to explain how the Government of Canada interacts with the public on social media platforms.",
+                  description: "Paragraph describing social media section",
+                })}
+              </p>
+              <p data-h2-margin-bottom="base(x3)">
+                {intl.formatMessage({
+                  id: "Tq0ZKw",
+                  defaultMessage:
+                    "Your engagement with the Government of Canada via social media is in part governed by the Terms of Service/Use of the relevant third-party social media platform providers, as well as the following Terms and Conditions. The Government of Canada has no control over the social media platform providers’ Terms of Service/Use, but you are strongly encouraged to read them in addition to those that follow.",
+                  description: "Paragraph describing social media section",
+                })}
+              </p>
+              <div
+                data-h2-display="base(flex)"
+                data-h2-flex-direction="base(column)"
+                data-h2-gap="base(x3)"
+              >
```

Comment:
Could we maybe try using margin on the headings to create the spacing instead? The `div` causes some really weird spacing. We rely on margin overlap for our typography spacing and these `div`'s negate all that 😢